README.md

```diff
@@ -1,8 +1,9 @@
 # LibreFanControl
 
+ [简体中文](./assets/zh_CN/README_zh-CN.md)
```

Comment:
Maybe it will make more sens to specify a keyword "README Translation" or something like that with a expand like that:
<details>
<summary>README Translation</summary>
简体中文
</details>

```
<details>
<summary>README Translation</summary>
简体中文
</details>
```

Reply:
Or a 3(or 4)-leveled Title?

### Translations
- [简体中文](./assets/zh_CN/README_zh-CN.md)